client/components/Overview/ImageGallery.jsx

```diff
@@ -44,7 +41,7 @@ const ImageGallery = (props) => {
               onClick={props.shrink}><AiOutlineCompress/></div>
 
             <img
-              style={{ marginLeft: ((-mousePosition.x) / 2) - 1000, marginTop: (-mousePosition.y) / 2 }}
+              style={{ marginLeft: ((-mousePosition.x) / 2) - 1000, marginTop: ((-mousePosition.y) / 2) }}
```

Comment:
This mouse tracking ability is really cool, I definitely look forward to work with something similar